components/script/dom/bindings/utils.rs

```diff
@@ -144,8 +153,8 @@ pub unsafe fn get_property_on_prototype(cx: *mut JSContext,
         return false;
     }
     *found = has_property;
-    let no_output = vp.ptr.is_null();
-    if !has_property || no_output {
+    // let no_output = vp.ptr.is_null(); // TODO can we remove?
```

Comment:
Yes, we can remove this one.

Reply:
Because this check was wrong to begin with (we should have been checking `vp.get.is_null()`), and it was removed from the equivalent upstream code two years ago.